test/component/front_end/ui/TaskListViewTest.java

```diff
@@ -0,0 +1,89 @@
+package component.front_end.ui;
+
+import static org.junit.Assert.*;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import entity.Task;
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+
+public class TaskListViewTest {
```

Comment:
Forgot to tell you in tests we don't use constants. Just write the magic values. Something like this:
```java
LinkedHashMap<String, String> commandWords = this.parser_.splitCommand("display:all task:\"Hello World!\" date");
assertThat(commandWords.get("display"), is(equalTo("all")));
assertThat(commandWords.get("task"), is(equalTo("Hello World!")));
assertThat(commandWords.containsKey("date"), is(true));
assertThat(commandWords.get("date"), is(nullValue()));
```